Main.java

```diff
@@ -9,7 +10,29 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+   // create a scanner for user input
+Scanner input = new Scanner(System.in);
+
+   //define a variable for the integer 3
+   final int A =  3;
+
+   //define a variable for the integer 5
+   final int B = 5;
+
```

Comment:
Once again, these should be input from the user, not hard coded in. Each of the samples shown is a different output that would be shown based on the input given.